static/js/src/chart-data.js

```diff
@@ -83,18 +83,36 @@ export var serverAndDesktopReleases = [
     taskName: "22.04 LTS (Jammy Jellyfish)",
     status: "ESM",
   },
-  {
-    startDate: new Date("2023-04-20T00:00:00"),
-    endDate: new Date("2024-01-20T00:00:00"),
-    taskName: "23.04 (Lunar Lobster)",
-    status: "INTERIM_RELEASE",
-  },
+  // {
```

Comment:
Please don't leave it commented like this, just remove it if it's not needed.
For the future, whenever you feel like you need to leave the code but commented, please add comments above explaining why you are leaving it.

Reply:
Updated, please check